src/components/MessagesList/MessagesGroup/Message/MessageButtonsBar/MessageForwarder.vue

```diff
@@ -131,7 +131,9 @@ export default {
 		},
 
 		openConversation() {
-			if (window.location.href.includes('/apps/files')) {
+			const isTalkApp = IS_DESKTOP || window.location.pathname.includes('/apps/spreed') || window.location.pathname.includes('/call/')
```

Comment:
'/apps/spreed' is included in the path of Talk "blank" page only so no MessageForwarder there. Unless I am missing another page with similar path :3 .

Reply:
No, just safe-checking that all Talk routes are included here